### ISSUE TITLE
[Ready for payment][$500] Chat - Skeleton in the header is not aligned with skeleton in the chat

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#35892]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v.1.4-37.0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause-Internal Team
**Slack conversation:**

## Action Performed:
1. Log out if logged in.
2. Log in to staging.new.expensify.com
3. Note that skeleton in the header.

## Expected Result:
The skeleton in the header has the same alignment to the left as the skeleton in the chat.

## Actual Result:
The skeleton in the header has extra margin on the left compared to the skeleton in the header.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

![Bug6369069_1707215412464!bandicam_2024-02-06_08-10-29-046](https://github.com/Expensify/App/assets/115492554/f98e4098-afc2-41c3-b6f6-3babf241ab60)

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01455b0defb7af246c</li>
        <li>Upwork Job ID: 1754825370662596608</li>
        <li>Last Price Increase: 2024-02-06</li>
<li>Automatic offers: </li>
<ul>
<li>bernhardoj | Contributor | 0</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @thesahindia</details>

**Comment by izarutskaya** [comment]
We think that this bug might be related to #vip-vsb 
CC @quinthar

**Comment by bernhardoj** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
The header skeleton has extra left space.

### What is the root cause of that problem?
The extra space is coming from the padding style here.
https://github.com/Expensify/App/blob/ff23077a0eb8b925281b048d0eae05391dd93162/src/pages/home/HeaderView.js#L239-L241

Previously, the padding is only added if the header is not loading, but after the ideal nav, specifically this [commit](https://github.com/Expensify/App/commit/73c6c75ee0d010ddde9e8f7034c75a9fede32286), the loading condition is removed.

### What changes do you think we should make in order to solve the problem?

Add the padding only if the header is loading.
```
!isSmallScreenWidth && isLoading && styles.pl5
```

**Comment by rlinoz** [comment]
Demoting, not a blocker.

**Comment by askavyblr** [comment]
Hello @izarutskaya 

I have Fix the issue , I am creating a PR 

Thanks 
![Screenshot from 2024-02-06 17-28-01](https://github.com/Expensify/App/assets/125897912/f23380f7-90db-40a0-ba85-fa8b1befece9)

**Comment by avnishjayaswal** [comment]
> 📣 @askavyblr! 📣 Hey, it seems we don’t have your contributor details yet! You'll only have to do this once, and this is how we'll hire you on Upwork. Please follow these steps:
> 
> 1. Make sure you've read and understood the [contributing guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md).
> 2. Get the email address used to login to your Expensify account. If you don't already have an Expensify account, create one [here](https://new.expensify.com/). If you have multiple accounts (e.g. one for testing), please use your main account email.
> 3. Get the link to your Upwork profile. It's necessary because we only pay via Upwork. You can access it by logging in, and then clicking on your name. It'll look like [this](https://www.upwork.com/freelancers/~01ca8ad8ce36b9eceb). If you don't already have an account, sign up for one [here](https://www.upwork.com/nx/signup/?dest=home).
> 4. Copy the format below and paste it in a comment on this issue. Replace the placeholder text with your actual details.
>    ![Screen Shot 2022-11-16 at 4 42 54 PM](https://user-images.githubusercontent.com/30609178/231263719-a8a3b7d3-36a0-4d7f-8ad4-cebe1f560fa4.png)
>    Format:
> 
> ```
> Contributor details
> Your Expensify account email: <REPLACE EMAIL HERE>
> Upwork Profile Link: <REPLACE LINK HERE>
> ```


Contributor details
Your Expensify account email: avnish.jayaswal@gmail.com
Upwork Profile Link: https://upwork.com/freelancers/avnishjaiswal

**Comment by askavyblr** [comment]
> 📣 @askavyblr! 📣 Hey, it seems we don’t have your contributor details yet! You'll only have to do this once, and this is how we'll hire you on Upwork. Please follow these steps:
> 
> 1. Make sure you've read and understood the [contributing guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md).
> 2. Get the email address used to login to your Expensify account. If you don't already have an Expensify account, create one [here](https://new.expensify.com/). If you have multiple accounts (e.g. one for testing), please use your main account email.
> 3. Get the link to your Upwork profile. It's necessary because we only pay via Upwork. You can access it by logging in, and then clicking on your name. It'll look like [this](https://www.upwork.com/freelancers/~01ca8ad8ce36b9eceb). If you don't already have an account, sign up for one [here](https://www.upwork.com/nx/signup/?dest=home).
> 4. Copy the format below and paste it in a comment on this issue. Replace the placeholder text with your actual details.
>    ![Screen Shot 2022-11-16 at 4 42 54 PM](https://user-images.githubusercontent.com/30609178/231263719-a8a3b7d3-36a0-4d7f-8ad4-cebe1f560fa4.png)
>    Format:
> 
> ```
> Contributor details
> Your Expensify account email: <REPLACE EMAIL HERE>
> Upwork Profile Link: <REPLACE LINK HERE>
> ```

Contributor details
Your Expensify account email: [avnish.jayaswal@gmail.com](mailto:avnish.jayaswal@gmail.com)
Upwork Profile Link: https://www.upwork.com/freelancers/~01826a49677e13626d

**Comment by askavyblr** [comment]
> Contributor details
> Your Expensify account email: [avnish.jayaswal@gmail.com](mailto:avnish.jayaswal@gmail.com)
> Upwork Profile Link: https://upwork.com/freelancers/avnishjaiswal

Contributor details
Your Expensify account email: avnish.jayaswal@expensify.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01826a49677e13626d

**Comment by askavyblr** [comment]
> ✅ Contributor details stored successfully. Thank you for contributing to Expensify

Hello 

I need to update my email ID in Contributor details what I need to do 

Thanks

**Comment by rlinoz** [comment]
@thesahindia @dukenv0307 should we handle this as a new issue or as a regression from that PR?

**Comment by dangrous** [comment]
@dukenv0307 can you restate what you said [here](https://github.com/Expensify/App/issues/35892#issuecomment-1930460836)? I'm not sure i'm understanding fully, and that would help us figure out if it's a regression or not - thanks!

**Comment by askavyblr** [comment]
> @askavyblr what do you want it updated to?
@rlinoz 
Thanks for your Reply
please update my email id to **avnish.jayaswal@gmail.com**

by mistake, I update the wrong email ID in the Contributor details

**Comment by c3024** [comment]
> @dukenv0307 can you restate what you said [here](https://github.com/Expensify/App/issues/35892#issuecomment-1930460836)? I'm not sure i'm understanding fully, and that would help us figure out if it's a regression or not - thanks!

There used to be a `!isLoading` check here 
https://github.com/Expensify/App/blob/38519c2e4e0d8774bbf6fcd10b2299372877f558/src/pages/home/HeaderView.js#L246
till this commit [here](https://github.com/Expensify/App/commit/38519c2e4e0d8774bbf6fcd10b2299372877f558)
and this check was removed 
https://github.com/Expensify/App/blob/45ec86749f9551d154bdd82727828e0f58e71620/src/pages/home/HeaderView.js#L230
from this commit [here](https://github.com/Expensify/App/commit/45ec86749f9551d154bdd82727828e0f58e71620).
This added the extra padding and caused this bug. These are unrelated to #35643 so I think this should not be considered as regression from #35643.

@dangrous @dukenv0307

**Comment by c3024** [comment]
https://github.com/Expensify/App/issues/35892#issuecomment-1936047010

@thesahindia

**Comment by dangrous** [comment]
Okay yeah I think I agree with that! So it was a mistake but not from #35643, it just interacted poorly with it

**Comment by rlinoz** [comment]
Cool!

@thesahindia, so I guess we can start looking into proposal's then.

**Comment by rlinoz** [comment]
Sounds good, lets go with @bernhardoj then!

Thanks!

**Comment by bernhardoj** [comment]
PR is ready

cc: @thesahindia

**Comment by rlinoz** [comment]
huumm the PR is in production already, the automation wasn't triggered I guess

@kadiealexander can you handle the payment on this one, please?

**Comment by bernhardoj** [comment]
@kadiealexander bump for the payment

**Comment by kadiealexander** [comment]
Sorry guys, this one slipped through the cracks!

**Comment by kadiealexander** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@thesahindia] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@thesahindia] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@thesahindia] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@thesahindia] Determine if we should create a regression test for this bug.
- [ ] [@thesahindia] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@kadiealexander] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by kadiealexander** [comment]
### Payment Summary
[Upwork Job](https://www.upwork.com/jobs/~01e4b313e99e0a0eca)
- Reviewer: @thesahindia owed $500 via NewDot
- Contributor: @bernhardoj owed 500 via Upwork

#### BugZero Checklist (@kadiealexander)
- [ ] I have verified the correct assignees and roles are listed above and updated the neccesary manual offers
- [ ] I have verified that there are no duplicate or incorrect contracts on Upwork for this job (https://www.upwork.com/ab/applicants/1772334661375188992/hired)
- [ ] I have paid out the Upwork contracts or cancelled the ones that are incorrect
- [ ] I have verified the payment summary above is correct

**Comment by kadiealexander** [comment]
Payment summary for Newdot payer [here](https://github.com/Expensify/App/issues/35892#issuecomment-2046430173).

**Comment by JmillsExpensify** [comment]
$500 approved for @thesahindia

**Comment by bernhardoj** [comment]
@kadiealexander hey, sorry, but I was checking my upwork and saw I haven't gotten paid for this one yet. The upwork offer link from [this comment](https://github.com/Expensify/App/issues/35892#issuecomment-2046430173) leads to this offer which I ended the contract just now because I don't work on https://github.com/Expensify/App/issues/38935, so I thought it's an offer error.

<img width="906" alt="image" src="https://github.com/Expensify/App/assets/50919443/e98eab78-88a4-4966-9874-60e26e651f7c">

I also see there is this [old offer](https://www.upwork.com/nx/wm/workroom/36307919) which is the correct one. Can you please recheck?

**Comment by kadiealexander** [comment]
Sorry about that @bernhardoj! I've paid this for you.

**Comment by bernhardoj** [comment]
Thanks!
